tdp/cli/commands/vars/vars_import.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import pathlib
+
+import click
+from exceptiongroup import ExceptionGroup
+from sqlalchemy import Engine
+
+from tdp.cli.params.collections_option import collections_option
+from tdp.cli.params.database_dsn_option import database_dsn_option
+from tdp.cli.params.validate_option import validate_option
+from tdp.cli.params.vars_option import vars_option
+from tdp.core.collections.collections import Collections
+from tdp.core.variables.cluster_variables import ClusterVariables
+from tdp.core.variables.import_vars import (
+    DEFAULT_VALIDATION_MESSAGE,
+    VALIDATION_MESSAGE_FILE,
+    import_vars,
+)
+from tdp.dao import Dao
+
+logger = logging.getLogger(__name__)
+
+
+@click.command("import")
+@click.argument(
+    "DIRECTORY",
+    envvar="TDP_OVERRIDES",
+    required=True,
+    type=click.Path(
+        exists=True, resolve_path=True, file_okay=False, path_type=pathlib.Path
+    ),
+    nargs=-1,
+)
+@click.option(
```

Comment:
Possible to have a `envvar` for this option ?

Reply:
Sth like `TDP_IMPORT_ERASE_BY_DEFAULT`?

Reply:
Done.